mbuild/tests/test_packing.py

```diff
@@ -16,11 +16,21 @@ def test_fill_box_density_box(self, h2o):
         filled = mb.fill_box(h2o, n_compounds=1000, density=1000)
         assert [3.1042931 < period < 3.1042932 for period in filled.periodicity]
 
+    def test_fill_box_aspect_ratio(self, h2o):
+        filled = mb.fill_box(h2o, n_compounds=1000,
+                density=1000, aspect_ratio=[1, 2, 1])
+        assert filled.periodicity[0]/filled.periodicity[1] == 0.5
+        assert filled.periodicity[1]/filled.periodicity[2] == 2
+
     def test_fill_box_density_n_compounds(self, h2o):
         filled = mb.fill_box(h2o, density=1000,
                              box=mb.Box([3.1042931, 3.1042931, 3.1042931]))
         assert filled.n_particles == 3000
 
+    def test_fill_box_compound_ratio(self, h2o, ethane):
+        filled = mb.fill_box(compound=[h2o, ethane], density=800,
+                compound_ratio=[2, 1], box=[2, 2, 2, 4, 4, 4])
```

Comment:
Can you also assert here that you indeed have placed h2o/ethane in the correct ratios?

Reply:
Well that test looks silly. But it would have failed anway if it that assertion existed. Now should be passing!